tasks/exec-utils.js

```diff
@@ -0,0 +1,12 @@
+import { exec, execSync as _execSync } from 'child_process';
+import { promisify } from 'util';
+
+const _execAsync = promisify(exec);
+
+export function execSync (command) {
+  return _execSync(command);
+}
+
+export function execAsync (command) {
+  return _execAsync(command);
+}
```

Comment:
note: sad to see Node.js not proposing Promise based versions of this

question: why don't we always use the `sync` version without any await in all the code?